pretty/src/pane.rs

```diff
@@ -12,44 +12,79 @@ where
     pub(crate) rect: Rect,
 }
 
+/// Specify the size of a subpane within a vertically or horizontally concatenated set of subpanes.
 #[derive(Clone, Copy, Debug)]
 pub enum PaneSize {
+    /// Give the subpane exactly this number of rows of height (for
+    /// `PaneNotation::Vert`) or columns of width (for `PaneNotation::Horz`).
     Fixed(usize),
+
+    /// After `Fixed` and `DynHeight` subpanes have been assigned a
+    /// width/height, divide up the remaining available width/height between the
+    /// `Proportional` subpanes according to their given weights. `Proportional`
+    /// subpanes with larger weights will be given more width/height.
```

Comment:
Replace last sentence with "The size of each subpane will be proportional to its weight, so that a subpane will weight 2 will be twice as large as one with weight 1, etc."